packages/synapse-react-client/src/components/HeaderCard.tsx

```diff
@@ -89,14 +95,27 @@ const HeaderCard: React.FunctionComponent<HeaderCardProps> = ({
                     </h3>
                   </div>
                   {subTitle && <div className="SRC-author"> {subTitle} </div>}
+                  {/* 
+                    Below is a hack that allows word highlighting to work, the Search component insert's
+                    html elements outside of the React DOM which if detected would break the app,
+                    but as written below this avoids that reconcilliation process.
+                  */}
                   {description && (
-                    <span className="SRC-font-size-base">
-                      {descriptionConfig?.isMarkdown ? (
-                        <MarkdownSynapse markdown={description} />
-                      ) : (
-                        description
-                      )}
-                    </span>
+                    <ShortDescription
+                      description={description}
+                      hasClickedShowMore={hasClickedShowMore}
+                      descriptionSubTitle=""
+                      descriptionConfig={descriptionConfiguration}
+                      toggleShowMore={() => setClickedShowMore(true)}
+                    />
+                  )}
+                  {description && (
+                    <LongDescription
+                      description={description}
+                      hasClickedShowMore={hasClickedShowMore}
+                      descriptionSubTitle=""
+                      descriptionConfig={descriptionConfiguration}
+                    />
```

Comment:
Looks good! If this replicates code in `GenericCard`, could consider pulling this into a new component (e.g. CollapsibleDescription?) that could be used by `GenericCard` and `HeaderCard`, but might be out of scope for this work. Up to you!